test/unit/api/define.js

```diff
@@ -3,18 +3,38 @@ import { Component, define, symbols } from '../../../src/index';
 const { name: $name } = symbols;
 
 describe('api/define', () => {
+  it('should not register without any properties', () => {
+    expect(() => {
+      define('x-test');
+    }).to.throw(Error);
+  });
+
+  it('should not register a component without a dash in the name', () => {
```

Comment:
I could be missing something but I don't think this would be testing any code paths in our codebase as this is up to the polyfill and native implementations.